src/plugin.js

```diff
@@ -235,6 +233,11 @@ const onPlayerReady = (player, options) => {
  */
 const errors = function(options) {
   this.ready(() => {
+    if (options) {
```

Comment:
Instead of modifying the error messages at the time the error is going to be displayed, you could go ahead and add the "You could also try installing Flash" right here. That would allow you to move `isFlashSupported` into this function, giving it a narrower scope, and get rid of `isCustomError` and your changes to the error handler entirely. In general, it is preferable to keep all the code that supports a particular behavior together in one place so I think the organization I'm suggesting would be better.